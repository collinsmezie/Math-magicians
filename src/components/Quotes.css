.quotesSection {
    width: 80%;
    margin: 13vh auto auto auto;
    font-size: 25px;
    text-align: center;
    font-weight: 200;
    color: rgb(72, 71, 68);
}