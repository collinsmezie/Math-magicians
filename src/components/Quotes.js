import React from 'react';
import '../component-styles/Quotes.css';

function Quotes() {
  return (
    <div className="quotesSection">
      &quot; Math magicians &quot; is a website for all fans of mathematics. It is a Single
      Page App (SPA) that allows users to make simple calculations and read a
      random math-related quote.
    </div>
  );
}

export default Quotes;
