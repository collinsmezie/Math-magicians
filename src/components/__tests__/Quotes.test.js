import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Quotes from '../Quotes';

describe('Quotes page of the App', () => {
  test('Builds the snapshot of the Quotes component safely', () => {
    const rend = renderer.create(<Quotes />);
    expect(rend.toJSON()).toMatchSnapshot();
  });
  test('Render Quotes component', () => {
    render(<Quotes />);
  });
});
