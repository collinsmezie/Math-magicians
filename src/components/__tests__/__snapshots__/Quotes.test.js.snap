// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes page of the App Builds the snapshot of the Quotes component safely 1`] = `
<div
  className="quotesSection"
>
  " Math magicians " is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
</div>
`;
